src/applications/combined-debt-portal/medical-copays/components/StatementTable.jsx

```
@@ -9,7 +9,7 @@ const StatementTable = ({ charges, formatCurrency, selectedCopay }) => {
   const MAX_ROWS = 10;
 
   function paginate(array, pageSize, pageNumber) {
-    return array.slice((pageNumber - 1) * pageSize, pageNumber * pageSize);
+    return array?.slice((pageNumber - 1) * pageSize, pageNumber * pageSize);
```

why would this reach this without an array? Feels like this might be a situation where an error would show up we'd want to see.

---

I would prefer that we remove this and catch it and raise an error. we should modify the mockdata to deal with this and not the other way around. As is, we could have silent errors passing through if paginate doesn't receive an array with actual items.

---

or do you not expect there to always be items in the array irl?